form.py

```diff
@@ -131,7 +140,8 @@ def review():
         cid = getCap()
 
         # get team members
-        mems = students.query.join(team_members).filter_by(tid=tid).distinct()
+        students = gbmodel.students()
+        mems = gbmodel.db_session.query(gbmodel.students).join(gbmodel.team_members).filter_by(tid=tid).distinct()
```

Comment:
This call doesn't seem guarded by a try-catch statement. If it isn't, is one needed here?

Reply:
fixed